core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java

```diff
@@ -17,32 +17,12 @@
     private boolean enabled;
 
     /**
-     * Defines value of failed-login-ratio metric on which an activation event will be generated. By default, the rule will be activated when 80% or
-     * more of the login requests will be a failed ones.
-     *
-     * Activation level unit are percentages. Metric failed-login-ratio is calculated as follows:
-     * <br/>
-     * <code>
-     *     failed-login-ratio = (100 * number or failed logins) / (number of all logins)
-     * </code>
-     * <br/>
-     * <br/>
-     *
-     * Activation and deactivation levels create hysteresis to better cope with credential stuffing and also to prevent too frequent
-     * activation/deactivation events.
+     * Percentage value of failed-login-ratio metric above which an activation event will be generated.
```

Comment:
Why this change? It would be good to have this in generated properties from script...

Reply:
As this is quite a low level implementation detail I think it would be better to keep it on the implementation itself, so I moved it to the FailedLoginRatioRule javadocs. 
Having it separately, on the properties, is prone to outdating when the implementation changes. Also bloating the properties documentation with such details would make the docs structure chaotic and not easy to follow - instead I'd rather kept the properties descriptions concise and created a dedicated section about the rules.

Reply:
It is actually quite important parameters that together with minimal sample size impact greatly on system response during credential stuffing.
It will be outdated anyway no matter when we would put it.

I preferred previous layout to be honest, properties can be viewed by more people theoretically, documenting on the class makes less sense as developers can just read the code.

Reply:
Ok, I see your point here. I restored the previous layout with some improvements and made the rule docs less detailed instead.